### PR TITLE
Added some descriptions in readme

### PR DESCRIPTION
**Pull request by genesisporridge**
Added some instructions on how to run subs tests with fault injection.

Some additional config in datastore and valkyrie.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ docker-compose up
 Valkyrie stubs may also be used as a testing library, for example by starting a genericpam server programmatically as
 in this Valkyrie test suite [example](https://github.com/valkyrie-fnd/valkyrie/blob/main/provider/internal/test/suite.go#L51).
 
+### Test configuration
+The stubs contain an in-memory database. To configure the database, edit the datastore [config file](./datastore.config.yaml). Make sure that the appropriate player, provider, provider API key and game are present in the config. Otherwise add the values.
+
+To configure the valkyre module for the test, make sure that the provider is present [here](./valkyrie_config.yml) too.
+
+*Note*: The config files mentioned in this section are included in the image in the docker build, so there is no point changing them in runtime. If stubs is built and run as a go binary, the config files should be present in the execution directory and can be updated between each stop and start.
```
Comment (gnirb) [inline README.md:57]: no point.. **in** changing them at runtime?

File: README.md [hunk 2]
```diff
@@ -57,15 +64,26 @@ There some predefined error scenarios can be triggered, like connection issues.
 
 In order to extend the available cases take a look at [scenarios.go](./broken/scenario.go).
 
+Once started, use the web interface to inject an appropriate error. If repeated errors (of the same kind) are wanted, start as many instances of the web interface as wanted and inject the fault once per instance.
```
Comment (gnirb) [inline README.md:67]: Maybe change "instances" to "tabs", otherwise it sounds like multiple broken instances to me

Comment (gnirb) [reply README.md:67]: This is perhaps something we should improve in the UI instead btw? 😄

**Review by gnirb — APPROVED** [review]
looks good to me!

File: README.md (hunk 1 above)
Comment (genesisporridge) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (genesisporridge) [reply README.md]: Done

**Comment by genesisporridge** [comment]
Adjusted instructions for testing with valkyrie-stubs/broken

**Pull request merged**